src/stores/connectors/stores/create_key.go

```diff
@@ -2,177 +2,78 @@ package stores
 
 import (
 	"context"
-	"time"
 
+	authtypes "github.com/consensys/quorum-key-manager/src/auth/entities"
+	"github.com/consensys/quorum-key-manager/src/auth/service/authorizator"
+	entities2 "github.com/consensys/quorum-key-manager/src/entities"
```

Comment:
idem

Reply:
entities are now split in the different domains but we have a tech debt to move them to a single folder: https://app.zenhub.com/workspaces/tanuki-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/431